extensions/ai-scorer/javascript/ai_scorer.js

```diff
@@ -7,7 +7,6 @@ const handleData = ($component) => {
       const image_data = JSON.parse(json);
 
       image_data.forEach((_data, index) => {
-        galleryData.set(`seed-${_data[`seed-${index}`]}-index-${index}`, _data);
```

Comment:
This is a bug where I previously did not remove this and should have been removed.  It doesn't cause the app to crash but it does throw annoying console errors.